guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to an archive file from {ProjectServer} in a syncable format.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+* RPM repositories
+* Kickstart repositories
+* File repositories
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the rpms, listing files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+
+To export a Content View, ensure that {ProjectServer} where you want to export meets the following conditions:
+
+* Ensure that you set download policy to *Immediate* for all repositories within the Content View you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize Products that you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+.To Export a Content View Version
+. List versions of the Content View that are available for export:
++
+[subs="+quotes"]
+----
+
+# hammer content-view version list \
+--content-view="_My_Content_View_" \
+--organization="_My_Organization_"
+
+---|----------|---------|-------------|-----------------------
+ID | NAME     | VERSION | DESCRIPTION | LIFECYCLE ENVIRONMENTS
+---|----------|---------|-------------|-----------------------
+5  | view 3.0 | 3.0     |             | Library
+4  | view 2.0 | 2.0     |             |
+3  | view 1.0 | 1.0     |             |
+---|----------|---------|-------------|----------------------
+
```

Comment:
```suggestion
```

We generally avoid interactive shell sessions throughout the docs.

Reply:
hmm i saw this kinda output here -> https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_exporting-a-content-view-version.adoc

I am open to removing from both if you are ok with it.

Reply:
Removing both sounds like a good idea to me.